x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/WaterfallItem.tsx

```diff
@@ -7,39 +7,66 @@
 import React from 'react';
 import styled from 'styled-components';
 
+import { EuiIcon } from '@elastic/eui';
 import {
   colors,
+  fontFamily,
   fontFamilyCode,
+  fontSize,
   fontSizes,
   px,
   unit,
   units
 } from '../../../../../../style/variables';
 import { IWaterfallItem } from './waterfall_helpers/waterfall_helpers';
 
-const ItemBar = styled.div`
+interface ItemBarProps {
+  type: 'transaction' | 'span';
+  left: number;
+  width: number;
+  color: string;
+}
+
+const ItemBar = styled<ItemBarProps, any>('div')`
+  box-sizing: border-box;
   position: relative;
-  height: ${unit}px;
+  height: ${props => (props.type === 'transaction' ? unit : unit)}px;
```

Comment:
I don't understand this change 😂 

```suggestion
  height: px(unit);
```

Reply:
lol was playing with something and didn't totally rip it out, good catch :banana: